### PR TITLE
fix: no_route found in local dev setup

### PR DESCRIPTION
**Pull request by bonomat**
We need to wait until the node announcement was processed correctly by lnd. This can be checked if lnd is aware of the coordinator's `node_alias`.

In the below patches we change the node announcement to 1/hour and allow triggering it manually using a http enpoint. With this I was so to always get into a working setup :)

fixes https://github.com/get10101/10101/issues/764

### CODE REVIEW

File: crates/ln-dlc-node/src/node/mod.rs [hunk 1]
```diff
@@ -7,6 +7,7 @@ use crate::ln::EventHandler;
 use crate::ln::TracingLogger;
```
Comment (holzeis) [inline crates/ln-dlc-node/src/node/mod.rs:7]: 🙃 the commit message actually says something different than what the change entails 😅 

"move helper function out of mod" would be correct 🙃

**Review by holzeis — APPROVED** [review]
Nice 👍

File: crates/ln-dlc-node/src/node/mod.rs (hunk 1 above)
Comment (bonomat) [reply crates/ln-dlc-node/src/node/mod.rs:7]: in-out, same same but different

**Pull request merged**